packages/autorest.python/autorest/codegen/models/operation.py

```diff
@@ -214,6 +214,14 @@ def default_error_deserialization(self) -> Optional[str]:
         # in this case, it's just an AnyType
         return "'object'"
 
+    @property
+    def has_special_status_code(self):
```

Comment:
I don't like having this here for a `mypy ignore`. I'm fine with coding this into `need_ignore` or some other property that explicitly deals with us needing to `mypy` ignore a line, but there is no significance for this property outside of mypy ignores

Reply:
Make sense.